datacapture/src/main/java/com/google/android/fhir/datacapture/views/factories/QuantityViewHolderFactory.kt

```diff
@@ -189,6 +191,11 @@ internal object QuantityViewHolderFactory :
             Coding(it.system, it.code, it.unit)
           }
             ?: questionnaireViewItem.draftAnswer?.let { if (it is Coding) it else null }
+              ?: questionnaireViewItem.questionnaireItem
+              .takeIf { it.hasInitial() }
+              ?.initialFirstRep
```

Comment:
shouldn't this just be 

```suggestion
              .initial
              ?.firstOrNull()
```

Reply:
or shouldn't you just call `questionnaireViewItem.questionnaireItem.unitOption.firstOrNull()`